pkg/registry/generic/etcd/etcd.go

```diff
@@ -403,6 +403,7 @@ func (e *Etcd) Delete(ctx api.Context, name string, options *api.DeleteOptions)
 			if lastGraceful > 0 {
 				return out, nil
 			}
+			return out, nil
 			// fall through and delete immediately
```

Comment:
Should probably remove this comment, and add one before this line that explains what is happening.  This block is basically now just `return out, nil` and that needs a comment.